core/src/main/kotlin/androidx/build/gradle/core/FileSystemStorageService.kt

```diff
@@ -86,7 +85,7 @@ internal class FileSystemStorageService(
 
     companion object {
         private fun File.deleteRecursively() {
-            val files = listFiles()
+            val files = listFiles() ?: return
```

Comment:
why would this return null?

Reply:
I mostly just did this because the IDE kept yelling at me tbh. Will revert